ruby/vampires.rb

```diff
@@ -1,11 +1,55 @@
-puts "what is your name?"
-name = gets.chomp
+puts "how many employees will be processed?"
+number = gets.chomp.to_i
 
-puts "how old are you? what you were you born?"
-age_year = gets.chomp
+survey = 0
+while survey < number
+
+  puts "what is your name?"
+  name = gets.chomp 
+    
+  puts "how old are you? what year were you born?"
+  age_year = gets.chomp.to_i
+  puts "should we order some garlic bread for you?"
+    garlic_bread = gets.chomp
+    
+  puts "would you like to enroll in the company's health insurance?"
+  health_insurance = gets.chomp
+  
+  puts "list out any allergies. type done when finished."
+  allergies = gets.chomp
+  
+    until allergies == "done"
+     puts "List out any allergies type done when finished."
+     allergies = gets.chomp
+    end
+  
+  if age_year < 100 && garlic_bread == "yes" || health_insurance == "yes"
+    	puts "Probably not a vampire"
+  elsif age_year > 100 && garlic_bread == "no" 
```

Comment:
It seems like we are using `age_year > 100` a lot.  Consider using the DRY (do not repeat yourself) method of design.  Could you assign this and other boolean results to a local variables?  If so you would only have to run that computation once and just use the variable wherever needed.